### DISCUSSION TITLE
Font size smaller on mobile when display flex

### DISCUSSION THREAD

**Discussion by arieleanarion** [twbs/bootstrap#40523] [Q&A]
Font size smaller on mobile than other text when display flex or display inline-block. Android Chrome 124

Visible on https://getbootstrap.com/docs/5.3/utilities/flex/#enable-flex-behaviors

(I'm a flexbox container!)

![scrnli_6_4_2024_3-12-05 PM](https://github.com/twbs/bootstrap/assets/99827419/a26bab90-ded2-4663-b169-e29857237d3a)

**Comment by mchubbit** [comment]
The font size issue on mobile might be related to how flexbox handles text scaling. Flex containers can sometimes cause text to render smaller, especially in certain browsers. You might find this article helpful for more insights: [Mobile App Development Services](https://ideamaker.agency/mobile-app-development-services/).
